arcgis-ios-sdk-samples/Analysis/Distance measurement analysis/DistanceMeasurementAnalysisViewController.swift

```diff
@@ -86,7 +86,7 @@ class DistanceMeasurementAnalysisViewController: UIViewController, AGSGeoViewTou
         sceneView.analysisOverlays.add(analysisOverlay)
         
         distanceView.backgroundColor = .backgroundGray
-        distanceView.tintColor = .primaryBlue
+        distanceView.tintColor = .primaryThemeColor
```

Comment:
This is the question I raised last week in our group chat with Mike. 

My observation is that:

- The `main.storyboard` file has a global tint color, which affects all the content display logic views in it. But this global tint is not inherited by any sample's separate storyboard.
![image](https://user-images.githubusercontent.com/9660181/89237767-ec114c00-d5a8-11ea-916b-bc4b490e689c.png)
- Our current setup makes every sample have a separate storyboard. If we do it by setting everything in storyboard, we'll need change every storyboard's (>180 samples 😓 ) global tint to our custom named theme color. I don't know if Xcode 12 can help with overriding them in one place. I'll read more about accent color to see if it helps.
- If we keep our current approach, i.e. 

    - overriding some `UIControl`s tint color in AppDelegate, e.g. tool bar, progress view, bar button, slider, etc.
    - set global tint for `main.storyboard`

  then we'll need to remove all other hardcoded tint colors like this from code, and make sure they get overriden. Still, this would require a few changes in some storyboards.

Reply:
Not every sample has a view that needs to be tinted. And for many that do, that is taken care of with the appearance changes made in the app delegate. For the remaining samples, setting the global tint color of the storyboard is a better solution than manually setting the tint color of certain views since it applies to every view in the sample. I'm not suggesting updating the global tint color of every storyboard. But for cases like this where we're setting it in code, we should switch to setting it in the storyboard itself. Does that make sense?

Reply:
Yes. This sounds like a more doable way to manage these colors. I'll make these changes later.

Reply:
Actually, after a closer examine to this particular hardcoded tint color, I find it is an invalid setting, since the `distanceView` itself is a stack view, so no color would ever take effect. I'll remove it altogether.

Reply:
Remember that `tintColor` is inherited from ancestors, so setting it on the stack view affects all subviews of it. That includes the segmented control. Since segmented controls are tinted on iOS 12 and we still support iOS 12, removing it altogether would make a difference.

Reply:
> Remember that `tintColor` is inherited from ancestors, so setting it on the stack view affects all subviews of it. That includes the segmented control. Since segmented controls are tinted on iOS 12 and we still support iOS 12, removing it altogether would make a difference.

OK I see. Then I'll still remove this view's hardcoded color, while set the global tint of its storyboard to accent color.